packages/blockchain-link-types/src/common.ts

```diff
@@ -188,6 +189,10 @@ export interface TokenInfo {
     accounts?: TokenAccount[]; // token accounts for solana
     policyId?: string; // Cardano policy id
     fingerprint?: string; // Cardano starting with "asset"
+    multiTokenValues?: MultiTokenValue[];
```

Comment:
Can you instead do 

```
 & Optional<Pick<BlockbookToken, 'multiTokenValues' | 'ids'>>;
```

Reply:
A lil different, but yes it's a nice idea

export interface TokenInfo extends Partial<Pick<Token, 'multiTokenValues' | 'ids'>> {